app.rb

```diff
@@ -0,0 +1,172 @@
+require_relative 'book'
+require_relative 'teacher'
+require_relative 'student'
+require_relative 'rental'
+require_relative 'classroom'
+
+class App
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def run
+    puts 'Welcome to School Library App!'
+    menu
+  end
+
+  def menu
+    loop do
+      puts ' '
+      puts 'Please choose an option by entering a number:'
+      puts '1 - List all books'
+      puts '2 - List all people'
+      puts '3 - Create a person'
+      puts '4 - Create a book'
+      puts '5 - Create a rental'
+      puts '6 - List all rentals for a given person id'
+      puts '7 - Exit'
+      print ''
+
+      choice = gets.chomp.to_i
+
+      case choice
+      when 1..6 then options(choice)
+      when 7 then puts 'Thank you for using our app.'
+      end
+      break if choice == 7
+    end
+  end
+
+  def options(choice)
+    case choice
+    when 1
+      list_books
+    when 2
+      list_people
+    when 3
+      create_person
+    when 4
+      create_book
+    when 5
+      create_rental
+    when 6
+      list_rentals
+    end
+  end
```

Comment:
- [ ] As mentioned by the previous :point_up: reviewer Please move this list of options into your main.rb file as it should deal with the user interactions only, all the methods should belong to the app class as you did already but still this code is there so kindly follow the instructions  because this is a requirement :slightly_smiling_face: 

**Requirements:**
- In your `main.rb` define the entry point, this will be a method called `main` that is invoked at the end of your file. 
  - This method should do the following:
  - Present the user with a list of options to perform.
  - Let users choose an option.
  - If needed, ask for parameters for the option.
  - Have a way to quit the app.

Reply:
gotcha. Implemented it. thank you.